application/views/registry_instrument_view.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+import structlog
+from flask import Blueprint, jsonify, make_response
+
+from application.controllers.basic_auth import auth
+from application.controllers.registry_instrument import RegistryInstrument
+
+log = structlog.wrap_logger(logging.getLogger(__name__))
+
+registry_instrument_view = Blueprint("registry_instrument_view", __name__)
+
+
+@registry_instrument_view.before_request
+@auth.login_required
+def before_registry_instrument_view():
+    pass
+
+
+@registry_instrument_view.route("/registry-instrument/exercise-id/<exercise_id>", methods=["GET"])
```

Comment:
Happy to debate the appropriateness of the name of this new resource. However, while implementing and trying different names I settled on `registry-instrument` and went through many different names.

Although similar to the existing named RASRM `collection-instrument`, there is no good way to name it while showing it's referring to a distinct external object.

Reply:
I think this name is appropriate, when reviewing a previous card i was struggling to think of a name for these resources and this seems an appropriate name for them.